cypress/integration/stoplight-project.ts

```diff
@@ -3,32 +3,121 @@ describe('Stoplight component', () => {
     cy.redirectUnpkgToLocalDistFolder('elements');
   });
 
-  it('loads correctly', () => {
-    loadStoplightProjectPage();
+  describe('Api page', () => {
+    it('loads api page correctly', () => {
+      loadStoplightProjectPage();
+      cy.findByText('To-dos').click();
+      cy.findByText('https://todos.stoplight.io/').should('exist');
+      cy.findByText('https://stoplight.io/mocks/elements-examples/studio-demo/389434').should('exist');
+      cy.findByRole('heading', { name: /API Key/ }).should('exist');
+      cy.findByText('💫 Overview').should('exist');
```

Comment:
Copy. I'd vote for removing this one, we probably won't lose much.

Reply:
Deleted